source/content/guides/composer-convert.md

```diff
@@ -92,18 +128,15 @@ A Composer-managed site should be able to include all custom code via Composer.
 
 When reviewing your site, take stock of exactly what versions of modules you depend on. One way to do this is to use a command like the following from within a contributed modules folder (e.g. `/modules`, etc):
```

Comment:
Here you can say "modules and themes", and it's fine to just say "One way to do this is to use a command like the following:" (it doesn't matter which folder you're in when you run drush pm:projectinfo)